src/main/java/frc/robot/commands/CameraTest.java

```diff
@@ -45,11 +45,20 @@ public void initialize() {
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    Optional<EstimatedRobotPose> robotPoseEstimator = m_camera.getFieldRelativePoseEstimator();
-    if (robotPoseEstimator.isEmpty()) return;
-    SmartDashboard.putString("Estimator Pose", robotPoseEstimator.get().estimatedPose.getTranslation().toString() + " | Rotation: " + Math.toDegrees(robotPoseEstimator.get().estimatedPose.getRotation().getAngle()*Math.PI));
+    Optional<EstimatedRobotPose> robotPoseEstimatorCam2 = m_photonCameras.getFieldRelativePoseEstimatorCam2();
```

Comment:
Should the command be coupled to there being two cameras? Maybe it can return a list of `EstimatedRobotPose`?